public/static/docs/command-reference/list.md

```diff
@@ -0,0 +1,138 @@
+# list
+
+List <abbr>repository</abbr> contents, including files and directories tracked
+by DVC (<abbr>data artifacts</abbr>) and by Git.
+
+## Synopsis
+
+```usage
+usage: dvc list [-h] [-q | -v] [-R] [--outs-only] [--rev [REV]] url [path]
+
+positional arguments:
+  url              Location of DVC repository to list.
+  path             Path to a file or directory within the repository.
+```
+
+## Description
+
+List files, dirs and <abbr>data artifacts</abbr> for the pointed URL. The output
+is sorted lexicographically.
```

Comment:
What do you mean exactly by "lexicographically" @JIoJIaJIu? I'm going to use "alphabetical order" for now.

Reply:
Yes, it means the same but more seriously :smile: